### PR TITLE
Feedback anatomy cleanups

### PR DESCRIPTION
**Pull request by halocline**
[Deploy Preview](https://deploy-preview-3089--keen-mayer-a86c8b.netlify.app/templates/user-feedback-collection#anatomy)

#### What does this PR do?

Proposed tweaks to anatomy implementation and copy.

Related to https://github.com/grommet/hpe-design-system/issues/2970

### CODE REVIEW

File: aries-site/src/pages/templates/user-feedback-collection.mdx [hunk 1]
```diff
@@ -29,8 +29,8 @@ Don’t collect feedback in the following use cases:
...
-| **1**  | **Title**    | Title prompting user for their feedback.            | <Checkmark a11yTitle="true" size="small" /> | "We'd love your feedback" is a nice, appreciative default title.       |
-| **2**  | **Body**     | The body may be populated as many questions as appropriate.          | <Checkmark a11yTitle="true" size="small" /> |  Question types can include StarRating, ThumbsRating, select all that apply (e.g. checkbox group), choose one (e.g. RadioButtonGroup), open ended (e.g. TextArea), or any other input types. |                                                             
+| **1**  | **Title**    | Title prompting user for their feedback.                             | <Checkmark a11yTitle="true" size="small" /> | <span>Use the [semantically correct heading level](/foundation/typography?q=semantically#semantic-usage-of-heading-levels) for the title. "We'd love your feedback" is a nice, appreciative default title.</span>       |
```
Comment (britt6612) [inline aries-site/src/pages/templates/user-feedback-collection.mdx:32]: one thing I wasnt sure about was the `required` should we put the checkmark even though its all required?

**Review by britt6612 — APPROVED** [review]

File: aries-site/src/pages/templates/user-feedback-collection.mdx (hunk 1 above)
Comment (halocline) [reply aries-site/src/pages/templates/user-feedback-collection.mdx:32]: I'm not following the question.

Comment (britt6612) [reply aries-site/src/pages/templates/user-feedback-collection.mdx:32]: if its all required do we still need this section for the table? I think of that as being overkill but my thought

Comment (halocline) [reply aries-site/src/pages/templates/user-feedback-collection.mdx:32]: Gotcha. Good to consider. I think we are okay here -- it'll be consistent with other anatomy tables.

**Pull request merged**